plugins/techdocs/api-report.md

```diff
@@ -326,12 +266,65 @@ const techdocsPlugin: BackstagePlugin<
 export { techdocsPlugin as plugin };
 export { techdocsPlugin };
 
-// Warning: (ae-missing-release-tag) "TechDocsReaderPage" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
-export const TechDocsReaderPage: ({
+// @public
+export const TechDocsReaderPage: (
+  props: TechDocsReaderPageProps,
+) => JSX.Element;
+
+// @public
+export const TechDocsReaderPageHeader: ({
+  entityRef,
+  entityMetadata,
+  techDocsMetadata,
   children,
-}: TechDocsPageProps) => JSX.Element;
+}: TechDocsReaderPageHeaderProps) => JSX.Element;
```

Comment:
reminder to unpack arguments inside the functions :) and in a couple other places - you can essentially search for `({` in this file

Reply:
haha oh I though I went through all of that, I'll take a look again 😆 Thanks!